README.md

```diff
@@ -1093,6 +1093,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Clair](https://askclair.ai/) - Clinical AI Reference. Students have free access to the professional tool suite, which includes Open Search, Clinical Summary, Med Review, Drug Interactions, ICD-10 Codes, and Stewardship. Additionally, a free trial for the professional suite is available.
   * [Langtrace](https://langtrace.ai) - enables developers to trace, evaluate, manage prompts and datasets, and debug issues related to an LLM application’s performance. It creates open telemetry standard traces for any LLM which helps with observability and works with any observability client. Free plan offers 50K traces/month.
   * [Ultra AI](https://ultraai.app) - AI command center for your product. Features include multi-provider AI gateway, prompts manager, caching, logs, analytics, model fallbacks and user rate limiting. Free forever plan offers 10k+ requests per month and all the features except caching.
+  * [Opik](https://github.com/comet-ml/opik) - Evaluate, test, and ship LLM applications across your dev and production lifecycles.
```

Comment:
This doesn’t appear to be a SaaS. While there is one called Comet its pricing isn’t made clear so don’t think this fits the list.

Reply:
Hi @ripienaar, thanks for the question! Comet is indeed the company behind Opik (the product name), which might get a bit confusing. You should be able to access pricing on this page: 

https://www.comet.com/site/pricing/

At the top, you should be able to toggle between "MLOps platform" (the original Comet ML product) and Opik (the new LLM evaluation product). 

If this isn't clear enough, let me know and I'll try to find something clearer! 🙂 

<img width="1354" alt="Screenshot 2024-09-27 at 7 38 15 AM" src="https://github.com/user-attachments/assets/4bc623b7-2ba6-4b4e-92c9-35af6dee255d">

Reply:
In that case comet is the free offering and what you should add here.